chirp/drivers/ft2d.py

```diff
@@ -74,9 +73,9 @@ class FT2D(ft1d.FT1Radio):
 
     _model = b"AH60M"  # Get this from chirp .img file after saving once
     _has_vibrate = True
-    _mem_params = (0x94a,          # Location of DTMF storage
-                   999,            # size of memories array
-                   999,            # size of flags array
+    _mem_params = (900,            # size of memories array
+                   900,            # size of flags array
+                   0xe4a,          # location of DTMF storage
```

Comment:
I haven't looked yet, but does anything need changing for the FT3? I have an FT3 and can test this later.

Reply:
FT2 and FT3 were faking access to the skip memories by claiming 999 memories (900 real-memory and 99 skip-memory; they're contiguous in the radio.) Re-factoring so Skip-memory as special allows return to 900 memories and associated flags.